skrub/_reporting/_data/templates/column-summaries.css

```diff
@@ -4,9 +4,18 @@
 /* Collection of column summary cards (as shown in the "column summaries" tab) */
 /* --------------------------------------------------------------------------- */
 
+/* Used as a container to enable horizontal scroll of the column
+ * summaries*/
+.horizontal-scroll {
+    overflow-x: scroll;
```

Comment:
you probably want `overflow-x: auto` because otherwise the scrollbar is always shown even when it is disabled because the screen is larger than the content (but anyway we can just use the existing horizontal-scroll class)

Reply:
OK, I thought that it was a good idea to always show it, but I think that you are right not to